src/v2/components/Channel/utils/throttle.ts

```diff
@@ -0,0 +1,21 @@
+import throttle from 'lodash/throttle';
+
+export const heavyThrottle = (
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  callback: (...params: any) => void,
+  ms = 2000,
+  option = {
+    leading: true,
+    trailing: true,
+  },
+) => throttle(callback, ms, option);
+
+export const lightThrottle = (
```

Comment:
This can be typed....

```
export const lightThrottle = <T extends (...args: any) => any>(
  callback: T,
  ms = 500,
  option: ThrottleSettings = {
    leading: true,
    trailing: true,
  },
) => throttle(callback, ms, option);
```

Reply:
Thanks for read-made types ;)